cpp/downgrades/19887dbd33327fb07d54251786e0cb2578539775/upgrade.properties

```diff
@@ -0,0 +1,4 @@
+description: Revert support for repeated initializers, which are allowed in C with designated initializers.
+compatibility: backwards
```

Comment:
What will the resulting downgraded database look like when a repeated initialiser occurs? With both tuples be present, and will the database just suffer from an inconsistency, or is one of the tuples dropped?

Reply:
Both tuples will be present and the database will suffer from an inconsistency.

Reply:
That's correct. I wasn't sure about how the compatibility categories apply to downgrade scripts. I'll change it to full.

Reply:
> I wasn't sure about how the compatibility categories apply to downgrade scripts

You're basically reasoning backwards as if the new and updated extractor is producing the old database and the old extractor is producing our fresh and exciting updated database.

I must admit I must always pause for a moment and think about this.

Reply:
Done. I'm assuming we don't need to rerun tests for this?

Reply:
No, I think that field is only informational.